src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -189,54 +163,6 @@ function trapClickOnNonInteractiveElement(node: HTMLElement) {
   node.onclick = emptyFunction;
 }
 
-function trapBubbledEventsLocal(node: Element, tag: string) {
```

Comment:
Sorry for not catching this earlier – I believe these need to stay here. In particular, if you do

```
<div onLoad={fn}>
<img />
</div>
```

then the div should receive the onLoad event (at least, that's what we currently do, whether or not that makes sense) even though the img itself doesn't have a listener. Listening at the div doesn't work since in the DOM, load doesn't bubble. I'm actually surprised we don't have a test for this.

Any chance you can try to add one (maybe just render that structure, then dispatch a load event to the img node) and then fix this? I think we will just have to leave the old structure unless you have ideas.